### PR TITLE
Edit UG

### PR DESCRIPTION
**Pull request by wnchan**
Standardise to use 'Case:' for all the expected outcome. Edit some spelling errors too.
Standardise for messages to be in ``.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -328,15 +252,14 @@ A student can edit to have any number of social media links. Add multiple social
...
+* Message: `Details edited successfully! Edited Student: [Updated data]`<br>
   ![sample result for 'edit'](images/edit.png)
 
 #### Expected Output (Failure):
-If no fields to edit are provided,
-* Message: “At least one field to edit must be provided.”
-
-If the email provided is not in the system,
-* Message: “Student with the provided email not found.”
+* Case: No fields provided for edit<br>
```
Comment (PearlynnT) [inline docs/UserGuide.md:259]:
```suggestion
* Case: No fields provided for edit.<br>
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -328,15 +252,14 @@ A student can edit to have any number of social media links. Add multiple social
...
+Message: `At least one field to edit must be provided.`
+* Case: Email not found in the system<br>
```
Comment (PearlynnT) [inline docs/UserGuide.md:261]:
```suggestion
* Case: Email not found in the system.<br>
```

**Review by PearlynnT — COMMENTED** [review]
Replacing the if sentences with case looks more readable now, thanks! Just missing some full stops

File: docs/UserGuide.md [hunk 3]
```diff
@@ -177,99 +178,23 @@ A student can have any number of social media links. Add multiple social media l
...
-* Major: Valid major names at NUS. More information listed [below](#valid-majors)
-* Year: Numeric year level. 
-* Email: Valid email address ending in “@u.nus.edu”. Maximum 20 characters.
+* Major: Valid major names at NUS. View the [appendix](#valid-majors) for more information.
+* Year: Numeric year level. Accept values between `1` and `6` inclusive.
+* Email: Valid email address ending in `@u.nus.edu`. Maximum 20 characters.
 * Description: Maximum 150 characters.
-* Tutorials: Two digit integers between 01 and 22 inclusive. Multiple tutorial slots can be added by using `t/` repeatedly.
+* Tutorials: Two digit integers between `01` and `22` inclusive. Multiple tutorial slots can be added by using `t/` repeatedly.
 * Social Media Link: Valid URL format to social media account. Multiple URLs can be added by using `sm/` repeatedly.
```
Comment (maj0-0) [inline docs/UserGuide.md:186]: Maybe can mention starting with http:// to be clear

File: docs/UserGuide.md [hunk 4]
```diff
@@ -808,3 +714,78 @@ _Details coming soon ..._
...
+| Sociology                                 |
+| South Asian Studies                       |
+| Southeast Asian Studies                   |
+| Statistics                                |
+| Systems Engineering                       |
+| Theatre Studies                           |
+| Urban Studies                             |
+| Visual Communications                     |
+
+If your major is not on this list, you can simply write "Others" as your major.<br>
```
Comment (maj0-0) [inline docs/UserGuide.md:789]: is it possible to put the Others as a code?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -808,3 +714,78 @@ _Details coming soon ..._
...
+
+If you have a double major or a major-minor, you are required to choose only one major. Please select any one of the majors listed above.
```
Comment (maj0-0) [inline docs/UserGuide.md:791]: this can included as a tip

**Review by maj0-0 — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 3 above)
Comment (PearlynnT) [reply docs/UserGuide.md:186]: it's https://

File: docs/UserGuide.md (hunk 1 above)
Comment (wnchan) [reply docs/UserGuide.md:260]: edited

File: docs/UserGuide.md (hunk 2 above)
Comment (wnchan) [reply docs/UserGuide.md:262]: edited.

File: docs/UserGuide.md (hunk 3 above)
Comment (wnchan) [reply docs/UserGuide.md:186]: edited

File: docs/UserGuide.md (hunk 4 above)
Comment (wnchan) [reply docs/UserGuide.md:791]: edited.

File: docs/UserGuide.md (hunk 5 above)
Comment (wnchan) [reply docs/UserGuide.md:793]: you mean to use tip format?

Comment (maj0-0) [reply docs/UserGuide.md]: if possible!

Comment (wnchan) [reply docs/UserGuide.md]: alright! edited!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -808,3 +716,82 @@ _Details coming soon ..._
 --------------------------------------------------------------------------------------------------------------------
 
 ## Appendix
+
```
Comment (neyapraveen) [inline docs/UserGuide.md:719]: thanks for moving the table!

File: docs/UserGuide.md [hunk 7]
```diff
@@ -691,29 +599,29 @@ Mark a task for a specified group as done.
...
-* Message: “Marked task number (ti) for group (gr) [and displays the updated task list]"
+* Message: `Marked task number (ti) for group (gr) [and displays the updated task list]`
   ![sample result for 'mark'](images/mark.png)
 
 #### Expected Output (Failure):
-* Case: Invalid mark command, e.g. 'mark ', 'mark t', etc.<br>
-Message: “Invalid command format! mark: Mark task specified as done. Parameters: gr/GROUP_NUMBER ti/TASK_INDEX Example: mark gr/2 ti/3”
-* Case: Invalid group number, a group number that is not found from the list in 'listGroup' command.<br>
-Message: “Group with the provided group number not found.”
+* Case: Invalid mark command, e.g. `mark`, `mark t`, etc.<br>
```
Comment (neyapraveen) [inline docs/UserGuide.md:615]: Using case looks good. Will update.

File: docs/UserGuide.md [hunk 8]
```diff
@@ -808,3 +716,82 @@ _Details coming soon ..._
...
+| South Asian Studies                       |
+| Southeast Asian Studies                   |
+| Statistics                                |
+| Systems Engineering                       |
+| Theatre Studies                           |
+| Urban Studies                             |
+| Visual Communications                     |
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+If your major is not on this list, you can simply write `Others` as your major.
```
Comment (Bearypop) [inline docs/UserGuide.md:792]: Maybe can move this to above the list so that it's more noticeable.

**Review by neyapraveen — COMMENTED** [review]
LGTM!

**Review by Bearypop — COMMENTED** [review]
Nice

File: docs/UserGuide.md [hunk 9]
```diff
@@ -808,3 +716,82 @@ _Details coming soon ..._
...
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+If you have a double major or a major-minor, you are required to choose only one major. Please select any one of the majors listed above.
```
Comment (PearlynnT) [inline docs/UserGuide.md:796]: double degree?

File: docs/UserGuide.md (hunk 8 above)
Comment (wnchan) [reply docs/UserGuide.md:792]: ok, will do.

File: docs/UserGuide.md (hunk 9 above)
Comment (wnchan) [reply docs/UserGuide.md:796]: i'll add that

**Pull request merged**